riscv/src/compiler.rs

```diff
@@ -1346,6 +1431,54 @@ fn process_instruction(instr: &str, args: &[Argument]) -> Vec<String> {
             only_if_no_write_to_zero(format!("{rd} <== load_label({label});"), rd)
         }
 
+        // atomic and synchronization
+        "fence" | "fence.i" => vec![],
+
+        insn if insn.starts_with("lr.w") => {
+            // Very similar to "lw", but without offset:
+            let (rd, rs) = rr(args);
+            // TODO misaligned access should raise misaligned address exceptions
```

Comment:
I guess that's gonna happen inside the memory SM for now?

Reply:
I just left a comment about the expected behavior, in case one day we decide to simulate RISCV hardware exceptions (which I guess we never will).